README_CMAKE.md

```diff
@@ -0,0 +1,53 @@
+You can now use CMake to build Arduino sketches with this core.
```

Comment:
CMake can now be used....

Passive form 😉

Reply:
Fixed!

Thanks for the feedback.